Load Image Pairs doesn't account for Use Limit

**Information:**

- Chainner version: Alpha 0.23.3
- OS: Win 11 pro
- 
**Description**
Not sure if this should be filed under bugs or feature requests. 
When loading image pairs, the use limit option is not taken into consideration. 
Example 1:
Dir A: 2023 images
Dir B: 1950 images
Use Limit: 1000

Both directories have 1000 images, this should supersede the total image count
Example 2:
If Dir A > Use Limit
and
Dir B > Use Limit
Dir A == Dir B 

**Logs**
Error

An error occurred in a Load Image Pairs node:

Number of images in directories A and B must be equal. Directory A: C:\%%%\Scu_Canv has 2972 images. Directory B: C:\%%%\2xNom__comp--2xNom_i_com\4x-ClearRealityV1 has 2889 images.

Input values:
• Directory A: "C:\\%%%\\Scu_Canv"
• Directory B: "C:\\%%%\\4x-ClearRealityV1"
• Use limit: Yes
• Limit: 1000
• Stop on first error: No

Stack Trace:
Traceback (most recent call last):
  File "C:\%%%\chaiNNer\app-0.23.3\resources\src\process.py", line 176, in run_node
    raw_output = node.run(*enforced_inputs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\%%%\chaiNNer\app-0.23.3\resources\src\packages\chaiNNer_standard\image\batch_processing\load_image_pairs.py", line 84, in load_image_pairs_node
    assert len(image_files_a) == len(image_files_b), (
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: Number of images in directories A and B must be equal. Directory A: C:\%%%\Scu_Canv has 2972 images. Directory B: C:\%%%\4x-ClearRealityV1 has 2889 images.

--- a/backend/src/packages/chaiNNer_standard/image/batch_processing/load_image_pairs.py
+++ b/backend/src/packages/chaiNNer_standard/image/batch_processing/load_image_pairs.py
@@ -81,15 +81,15 @@ def load_image_pairs_node(
     image_files_a: list[Path] = list_all_files_sorted(directory_a, supported_filetypes)
     image_files_b: list[Path] = list_all_files_sorted(directory_b, supported_filetypes)
 
+    if use_limit:
+        image_files_a = image_files_a[:limit]
+        image_files_b = image_files_b[:limit]
+
     assert len(image_files_a) == len(image_files_b), (
         "Number of images in directories A and B must be equal. "
         f"Directory A: {directory_a} has {len(image_files_a)} images. "
         f"Directory B: {directory_b} has {len(image_files_b)} images."
     )
-
-    if use_limit:
-        image_files_a = image_files_a[:limit]
-        image_files_b = image_files_b[:limit]
 
     image_files = list(zip(image_files_a, image_files_b))
 

